### PR TITLE
配置中心增加对客户端暴露 GRPC 协议的能力

### PR DESCRIPTION
**Pull request by lepdou**
**Please provide issue(s) of this PR:**

1. 配置中心增加对客户端暴露 GRPC 协议的能力
2. ConnectionManager 模块从 apiserver 下沉到 config server

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [x] Configuration
- [x] Docs
- [ ] Installation
- [ ] Performance and Scalability
- [ ] Naming
- [ ] HealthCheck
- [ ] Test and Release

**Please check any characteristics that apply to this pull request.**

- [x] Does not have any user-facing changes. This may include API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

**Comment by lepdou** [comment]
implement #314

File: README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ https://dev.mysql.com/downloads/mysql/5.7.html
 
 #### Import SQL script
 
-Point Script: ./store/sqldb/polaris_server.sql, one can import through mysql admin or console.
+Point Script: ./store/sqldb/scripts/polaris_server.sql, one can import through mysql admin or console.
```
Comment (andrewshan) [inline README.md:79]: mysqladmin，这是一个工具名称吧？不适合拆开写

File: common/api/v1/codeinfo.go [hunk 2]
```diff
@@ -154,7 +154,6 @@ const (
 	InvalidConfigFileFormat        = 400804
 	InvalidConfigFileTags          = 400805
 	InvalidWatchConfigFileFormat   = 400806
-	NotFoundResourceConfigFile     = 400807
```
Comment (andrewshan) [inline common/api/v1/codeinfo.go:157]: 配置不存在的话，现在不返回错误码么？

**Review by andrewshan — APPROVED** [review]
OK

**Pull request merged**